config/locales/en.yml

```diff
@@ -818,7 +818,7 @@ en:
     title:
       episode_image: Cover Image
       feed_image: Thumbnail Image
-      itunes_image: Profile Image
+      itunes_image: Cover Image
```

Comment:
Its a little strange that there are three labels defined here, but only two use cases. Regardless, this is the quick fix.

Reply:
I think all 3 are needed, since we infer the title from the [model name](https://github.com/PRX/feeder.prx.org/blob/main/app/views/images/_new.html.erb#L4).